### PR TITLE
Reinstate view top account balances

### LINKED ISSUE #4674: Allow users to select token to view top account balance
### Description

When a blockchain application supports more than one token (e.g DEX blockchain application), users should have the ability to choose/select the token to view the top accounts (by highest balances for the selected token)
 
### Acceptance Criteria

- By default show top accounts and its balance based on endpoint
- Users should be able to select token (if applications supports more than 1 token)
  - If the blockchain application has only 1 token supported, then do not show the token selection
- Add unit test
- Add e2e test
 
### Additional Information

https://github.com/LiskHQ/lisk-service/issues/1383

- To implement the supply use total supply to calculate the percentage https://petstore.swagger.io/?url=https://betanet-service.lisk.com/api/v3/spec#/Token/get_token_summary

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #4674

### How was it solved?

- Reinstating page view and implementing new [design](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=2-7&mode=design&t=yyJRXQjjTwEQwH4J-0)

### How was it tested?

- [x] Visually
- [x] Unit tests

### CODE REVIEW

File: setup/jest.config.js [hunk 1]
```diff
@@ -319,6 +319,7 @@ module.exports = {
     'src/modules/blockchainApplication/manage/components/UserApplicationSelector/UserApplicationSelector.js', // TODO: To be fixed in PDD-1522
     'src/modules/network/components/selectNetwork/SelectNetwork.js', // TODO: To be fixed in PDD-1522
     'src/modules/blockchainApplication/manage/components/NetworkApplicationDropDownButton/NetworkApplicationDropDownButton.js', // TODO: To be fixed in PDD-1522
+    'src/modules/wallet/components/walletList/tableHeader.js',
```
Comment (ManuGowda) [inline setup/jest.config.js:321]: why skipping?

File: src/modules/common/components/bars/sideBar/menuLinks.js [hunk 2]
```diff
@@ -40,12 +46,6 @@ const menuLinks = (t) => [
       label: t('Network'),
       path: routes.network.path,
     },
-    // {
```
Comment (ManuGowda) [inline src/modules/common/components/bars/sideBar/menuLinks.js:43]: Remove this

File: src/modules/token/fungible/mocks/index.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const tokensBalance = rest.get(
 );
 
 export const tokensTopLskBalance = rest.get(
```
Comment (ManuGowda) [inline src/modules/token/fungible/mocks/index.js]:
```suggestion
export const tokenBalancesTop` = rest.get(
```

File: src/modules/wallet/components/row/wallet.css [hunk 4]
```diff
@@ -6,3 +6,15 @@
 .balanceShare {
   white-space: nowrap;
 }
+
+.accountSummary {
```
Comment (ManuGowda) [inline src/modules/wallet/components/row/wallet.css]: you renamed all places account to wallet and here why the usage of account?

File: src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js [hunk 5]
```diff
@@ -18,14 +18,14 @@ import { useCustomInfiniteQuery } from 'src/modules/common/hooks';
 
 export const useTokensTopLskBalance = ({ config: customConfig = {}, options } = {}) => {
```
Comment (ManuGowda) [inline src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js]:
```suggestion
export const useTokenBalancesTop = ({ config: customConfig = {}, options } = {}) => {
```

Comment (ManuGowda) [reply src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js]: Please rename the file as well `src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js`

File: src/modules/wallet/utils/helpers.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+export const calculateSupply = (balance, totalSupply) =>
```
Comment (ManuGowda) [inline src/modules/wallet/utils/helpers.js]: If its not reused can we move this the place where its only being used?

File: src/modules/common/components/bars/sideBar/menuLinks.js [hunk 7]
```diff
@@ -28,6 +28,12 @@ const menuLinks = (t) => [
       label: t('Blocks'),
       path: routes.blocks.path,
     },
+    {
```
Comment (ManuGowda) [inline src/modules/common/components/bars/sideBar/menuLinks.js]: Place it after `validatorsMonitor`

File: setup/jest.config.js (hunk 1 above)
Comment (ikem-legend) [reply setup/jest.config.js:321]: This is usually covered by end-to-end tests for the monitor view hence the skipping

File: src/modules/common/components/bars/sideBar/menuLinks.js (hunk 2 above)
Comment (ikem-legend) [reply src/modules/common/components/bars/sideBar/menuLinks.js:43]: Done

File: src/modules/common/components/bars/sideBar/menuLinks.js (hunk 7 above)
Comment (ikem-legend) [reply src/modules/common/components/bars/sideBar/menuLinks.js]: Done

File: src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js (hunk 5 above)
Comment (ikem-legend) [reply src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js]: Done

File: src/modules/token/fungible/mocks/index.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/token/fungible/mocks/index.js]: DOne

File: src/modules/wallet/utils/helpers.js (hunk 6 above)
Comment (ikem-legend) [reply src/modules/wallet/utils/helpers.js]: Fixed

File: src/modules/wallet/components/row/wallet.css (hunk 4 above)
Comment (ikem-legend) [reply src/modules/wallet/components/row/wallet.css]: Fixed

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/account/components/Overview/overview.js [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    setFilter('tokenID', tokenData.tokenID);
+  };
+
+  return (
+    <>
+      <div>
+        <h1>{t('All accounts')}</h1>
+      </div>
+      <div className={styles.filterWrapper}>
+        <span>Filter by:</span>
```
Comment (eniolam1000752) [inline src/modules/account/components/Overview/overview.js]: Figma as says this

```suggestion
        <span>Filter by :</span>
```

File: src/modules/account/components/Overview/overview.js [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+        <MenuSelect
+          value={selectedToken}
+          onChange={onChange}
+          select={(selectedValue, option) => selectedValue?.tokenName === option.tokenName}
+          className={styles.menuWrapper}
+          popupClassName={styles.popupWrapper}
+        >
+          {applicationTokens.map((tokenValue) => (
```
Comment (eniolam1000752) [inline src/modules/account/components/Overview/overview.js]: `tokenValue` seems confusing there can it be `token`

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
## UI/UX issues

- Loading state on mount not properly handled. The UI of the dropdown looks weird on first mount
image

- Also the UI of an account without any token  looks dry. I think there should be an empty state enabling a user to request a token.
image

- The height of the search bar is not corresponding with the height of the token select dropdown. Also the search icon has an offset and is not properly horizontally centralised
image

- Inputing a any search text fails with a 400 error. 
image

- I clicked on a particular account and went to its wallet page. I also tried initiating a send transaction but on inputing 9 as the token value I am weirdly getting an insufficient balance error. Which is very weird.
image

- On toggling sorting on a particular column, the sort direction arrow does not react nor change. Normally if i sort in ascending order, the arrow should point to the bottom and vise-versa.
image

- The sort logic is kind of weird. When I toggle the sort operation on balance, I get a lesser number of account rendered. Then when I toggle supply, I get a larger number of account rendered.

- Figma says there should be no margin but on the implementation there is a margin of 13px
image

- Figma says **_Rank_** should be sortable but it was not implemented
image

-

File: src/modules/account/components/Overview/overview.js (hunk 9 above)
Comment (ikem-legend) [reply src/modules/account/components/Overview/overview.js]: Fixed

File: src/modules/account/components/Overview/overview.js (hunk 8 above)
Comment (ikem-legend) [reply src/modules/account/components/Overview/overview.js]: Fixed

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
- Search Icon is still not centre aligned
image

**Review by ManuGowda — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**